docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -113,10 +157,93 @@ Use the following code snippet to change the default settings:
          ov::AnyMap pipeline_config = { { "MAX_PROMPT_LEN",  1024 }, { "MIN_RESPONSE_LEN", 512 } };
          ov::genai::LLMPipeline pipe(model_path, "NPU", pipeline_config);
 
+Cache compiled models
++++++++++++++++++++++
+
+Specify the ``NPUW_CACHE_DIR`` option in ``pipeline_config`` for NPU pipeline to
+cache the compiled models. Using the code snippet below shortens the initialization time
+of the pipeline runs coming next:
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+      .. code-block:: python
+
+         pipeline_config = { "NPUW_CACHE_DIR": ".npucache" }
+         pipe = ov_genai.LLMPipeline(model_path, "NPU", pipeline_config)
+
+   .. tab-item:: C++
+      :sync: cpp
+
+      .. code-block:: cpp
+
+         ov::AnyMap pipeline_config = { { "NPUW_CACHE_DIR",  ".npucache" } };
+         ov::genai::LLMPipeline pipe(model_path, "NPU", pipeline_config);
+
+
+Disable memory allocation
++++++++++++++++++++++++++
+
+In case of execution failures, either silent or with errors, try to update the NPU driver to
+`newer than 31.0.100.3053 <https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html>`__.
```

Comment:
There's no 31.* drivers. It is only 32.*
I believe it should sound as "update to *.3104 or newer".

Reply:
> There's no 31.* drivers. It is only 32.* I believe it should sound as "update to *.3104 or newer".

![image](https://github.com/user-attachments/assets/cdf453a2-7424-4d56-9cf6-6c59f0c10a15)

By ***.3104** or newer, you meant "32.0.100.2196 or newer" ?

Reply:
Interesting what that 31.0.100 thing is - nevermind :)

Anyways we're waiting for 3104 to be published - so it must be 3104 or newer.

Reply:
How about:
```suggestion
`the lastest version <https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html>`__ (**other** than 31.0.100.3053) .
```
Once `.3104` is out we can change the link accordingly. What do you say?